control/tests/robust_array_test.py

```diff
@@ -1,393 +0,0 @@
-import unittest
```

Comment:
Same code as robust_test.py, just different setup and teardown. So obsolete with the new fixture.